lib/OrdinaryDiffEqSSPRK/src/algorithms.jl

```diff
@@ -320,7 +320,16 @@ function SSPRK33(stage_limiter!, step_limiter! = trivial_limiter!)
         step_limiter!, False())
 end
 
-@doc explicit_rk_docstring("TBD", "SHLDDRK_2N")
+@doc explicit_rk_docstring("Low dissipation and dispersion Runge-Kutta schemes for computational acoustics", "SHLDDRK_2N",
```

Comment:
SDIRK missing stuff @ChrisRackauckas 

Reference:
ImplicitEuler
ImplicitMidpoin

Description:
ESDIRK436L2SA2
ESDIRK437L2SA
ESDIRK547L2SA2
ESDIRK659L2SA

Reference and description:
SDIRK22
SSPSDIRK2
CFNLIRK3
SFSDIRK4 - 8
ESDIRK54I8L2SA

Reply:
SSPSDIRK is an SSP-optimized SDIRK method, so it's an implicit SDIRK method for handling stiffness but if the `dt` is below the SSP coefficient * dt, then the SSP property of the SSP integrators (the other page) is satisified. https://www.davidketcheson.info/assets/papers/implicit_ssp_postprint.pdf As such this is a method which is expected to be good on advection-dominated cases where an explicit SSP integrator would be used, but where reaction equations are sufficient stiff to justify implicit integration.

Reply:
SFSDIRK methods are https://www.sciencedirect.com/science/article/pii/S0168927407001651 "with special nonlinear stability properties indicated by the terms total-variation-diminishing (TVD), strongly stable and monotonic".

Reply:
https://github.com/SciML/OrdinaryDiffEq.jl/pull/1113

Reply:
CFNLIRK3  is the third order method of https://github.com/SciML/OrdinaryDiffEq.jl/issues/1093 https://www.sciencedirect.com/science/article/pii/S0168927400000611

Reply:
SDIRK22 is from https://ntrs.nasa.gov/api/citations/20160005923/downloads/20160005923.pdf